doc/source/acb.rst

```diff
@@ -1162,6 +1177,22 @@ Vector functions
 
     Returns nonzero iff all entries in *x* have zero imaginary part.
 
+.. function:: int _acb_vec_is_finite(acb_srcptr vec, slong len)
+
+    Returns nonzero iff all entries in *x* certainly are finite.
+
+.. function:: int _acb_vec_equal(acb_srcptr vec1, acb_srcptr vec2, slong len)
```

Comment:
Maybe clarify that this is equal-as-intervals in the sense of ``acb_equal`` and not equal-as-numbers in the sense of ``acb_eq``. And likewise for the ``arb`` versions.

Reply:
This is fixed in https://github.com/flintlib/flint/pull/1524/commits/e4cafd2fa5f95468b60811afc5d8050b57b21ef2.